articles/users/search.md

```diff
@@ -302,7 +306,84 @@ When you create your job, you'll need to provide:
 }
 ```
 
-Once you've created your job to export your users, you can check on its status using the [Get a Job endpoint](/api/management/v2#!/Jobs/get_jobs_by_id). You'll need to provide the ID of the job (which you received in the response when creating the job) -- if you're using the sample request below, replace the placeholder `YOUR_JOB_ID` with the value of the ID.
+### Export metadata
+
+If you export in CSV and include metadata information, you have to specify each field that you want exported. 
```

Comment:
If you export user data to a CSV file and want to include metadata information, you will need to specify each (metadata) field you want exported.